holoviews/streams.py

```diff
@@ -223,3 +223,60 @@ class PositionXY(Stream):
     def __init__(self, preprocessors=[], source=None, subscribers=[], **params):
         super(PositionXY, self).__init__(preprocessors=preprocessors, source=source,
                                         subscribers=subscribers, **params)
+
+
+
+
+class ParamValues(Stream):
+    """
+    A Stream based on the parameter values of some other parameterized
+    object, whether it is a parameterized class or a parameterized
+    instance.
+
+    The update method enables the stream to update the parameters of the
+    specified object.
+    """
```

Comment:
Well, in the case of paramnb the widgets set the parameter values directly and the update method won't really do anything, so there's two ways the parameters can be set. HoloViews doesn't care which one is actually used though.